frontend/src/scenes/persons/Person.tsx

```diff
@@ -207,6 +208,14 @@ export function Person(): JSX.Element | null {
                         <RelatedGroups id={person.uuid} groupTypeIndex={null} />
                     </TabPane>
                 )}
+                {person.uuid && (
+                    <TabPane
+                        tab={<span data-attr="persons-related-flags-tab">Feature flags</span>}
+                        key={PersonsTabType.FEATURE_FLAGS}
+                    >
+                        <RelatedFeatureFlags distinctId={person.uuid} />
```

Comment:
this needs to be a distinctID, not the uuid. Giving wrong results right now 😅

Reply:
distinctId comes in as an array, do you want me to just grab the first one from the array or?

Reply:
Added it as the first option from the array, can change when you're back from haircut if it's wrong 😛

Reply:
lol, yeah, good enough!